### ISSUE TITLE
[Bug] Create's Sequenced Assembly Recipes cannot open

### ISSUE THREAD

**Issue by NatsumiXD** [shedaniel/RoughlyEnoughItems#1710]
### What happened?

I cannot see anything about the Sequenced Assembly Recipes in the REI

### What mod loaders are you seeing the problem on?

Forge

### What do you think this bug is of?

- [ ] Visual
- [X] Recipe Lookup
- [ ] Cheat Mode
- [X] Plugin Integration / JEI Plugin Compatibility
- [ ] Others

### Relevant log output

```shell
later
```

### Anything else?

This seems to be a bug caused by randomness

### By submitting this issue, I have included the necessary logs by pasting the contents into the correct location or attaching the file as an upload.

- [X] Yes, and I did not use any paste services other than GitHub Gists.

### By submitting this issue, I have confirmed my REI and REI's dependencies are up to date.

- [X] Yes

**Comment by NatsumiXD** [comment]
that is the log
[latest1.log](https://github.com/user-attachments/files/16669382/latest1.log)

**Comment by NatsumiXD** [comment]
[rei1.log](https://github.com/user-attachments/files/16669410/rei1.log)

**Comment by Barerock** [comment]
Yep. Happens with Applied Kinetics and Protection Pixel especially for me. Even when their recipes are 100% disabled and replaced with KubeJS versions, the items themselves lose their recipe AND tags in REI, so pressing R or U does nothing. This is fixed if a shaped or otherwise NOT sequenced assembly recipe is made for the items. I tested this with Applied Kinetics mostly, but noticed similar behavior with Protection Pixel today. Sometimes the recipes will show, sometimes they won't; this happens to be independent of /reloadconfig, and will randomly occur in a session either way. Usually if they appear, then disappear, they will not reappear again until a new session is made. Specifically the Applied Kinetics sequenced assemblies for the printed processors and processors are broken, and the -AS armor in Protection Pixel. As well as, I can assume, the alloy armor plate in Protection Pixel, which I gave a shaped recipe before I noticed the other stuff misbehaving.

Glad to know I'm not crazy. Seems like mods that use sequenced assembly that use mods not from or adjacent to Create tend to be buggy like this. This does not happen with TFMG, or other Create addons. I guess a place to start is looking at how those mods implement sequenced assembly vs these other two to compare where REI is being broken.

Amendment: This _does_ happen with TFMG, but I didn't think so because it's only like 5 items. I know for a fact that sequenced assemblies from Create work, I'm looking at one right now for Sturdy Sheet. So it seems like mods adding sequenced assembly recipes break when loaded through REI. I hope some of this pinpointed whatever for you guys.

**Comment by dudebehinddude** [comment]
I can confirm that this been and still is an issue

**Comment by LMaxRouterCN** [comment]
> I can confirm that this been and still is an issue

Yes, until now.
